src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(numbers) {
+    if (numbers === undefined) {
```

Comment:
Strictly comparing 'numbers' to 'undefined' might lead to unexpected results if 'null' or other falsy values are passed. Consider using a type check like 'typeof number === 'undefined'' for clarity.